tools/scripts/aducm.mk

```diff
@@ -133,6 +133,9 @@ HEX = $(basename $(BINARY)).hex
 #                                 RULES                              
 #------------------------------------------------------------------------------
 
+PHONY += hex
+hex: $(HEX)
```

Comment:
hex: is only available for aducm and this creates a nightmare when documenting, we need to keep track of all platform-specific make rules and document each separately. Instead, we should converge on a set of universal make rules for all platforms. This could be part of a postprocessing: or postbuild: step that every platform .mk may implement or not (optional)

Reply:
done